static/panes/cfg-view.ts

```diff
@@ -22,427 +22,323 @@
-            searchField: ['name'],
+export class Cfg extends Pane<CfgState> {
+    graphDiv: HTMLElement;
+    svg: SVGElement;
+    blockContainer: HTMLElement;
+    graphContainer: HTMLElement;
+    graphElement: HTMLElement;
+    infoElement: HTMLElement;
+    currentPosition: Coordinate = {x: 0, y: 0};
+    dragging = false;
+    dragStart: Coordinate = {x: 0, y: 0};
+    dragStartPosition: Coordinate = {x: 0, y: 0};
+    graphDimensions = {width: 0, height: 0};
+    functionSelector: TomSelect;
+    results: CFGResult;
+    state: CfgState & PaneState;
+    layout: GraphLayoutCore;
+    bbMap: Record<string, HTMLDivElement> = {};
+
+    constructor(hub: Hub, container: Container, state: CfgState & PaneState) {
+        if ((state as any).selectedFn) {
+            state = {
+                id: state.id,
+                compilerName: state.compilerName,
+                editorid: state.editorid,
+                treeid: state.treeid,
+                selectedFunction: (state as any).selectedFn,
+                zoom: 1,
+            };
+        }
+        super(hub, container, state);
+        this.eventHub.emit('cfgViewOpened', this.compilerInfo.compilerId);
+        this.eventHub.emit('requestFilters', this.compilerInfo.compilerId);
+        this.eventHub.emit('requestCompiler', this.compilerInfo.compilerId);
+        const selector = this.domRoot.get()[0].getElementsByClassName('function-selector')[0];
+        if (!(selector instanceof HTMLSelectElement)) {
+            throw new Error('.function-selector is not an HTMLSelectElement');
+        }
+        this.functionSelector = new TomSelect(selector, {
+            valueField: 'value',
+            labelField: 'title',
+            searchField: ['title'],
             dropdownParent: 'body',
             plugins: ['input_autogrow'],
-            onChange: (e: any) => {
-                // TomSelect says it's an Event, but we receive strings
-                const val = e as string;
-                if (val in this.functions) {
-                    const selectedFn = this.functions[val];
-                    this.currentFunc = val;
-                    this.showCfgResults({
-                        nodes: selectedFn.nodes,
-                        edges: selectedFn.edges,
-                    });
-                    if (selectedFn.nodes && selectedFn.nodes.length > 0) {
-                        this.cfgVisualiser.selectNodes([selectedFn.nodes[0].id]);
-                    }
-                    this.resize();
-                    this.updateState();
-                }
+            sortField: 'title',
+            onChange: e => {
+                this.selectFunction(e as any as string);
             },
         });
-
-        this.updateButtons();
+        this.state = state;
     }
 
-    override getInitialHTML(): string {
+    override getInitialHTML() {
         return $('#cfg').html();
     }
 
+    override getDefaultPaneName() {
+        return 'CFG';
```

Comment:
As I said in dc, Cypress is failing because of this change, as they are suuuper simple tests that only check for the pane name to exists in rder to say "Yes, this works". Changing https://github.com/compiler-explorer/compiler-explorer/blob/main/cypress/integration/frontend-testing.js#L21 to `CFG` should fix it